src/states/elder/mod.rs

```diff
@@ -335,7 +328,7 @@ impl Elder {
             }
         }
         if let Some(merged_info) = self.chain.try_merge()? {
-            self.vote_for_section_info(merged_info)?;
+            self.vote_for_signed_event(merged_info)?;
```

Comment:
No response to https://github.com/maidsafe/routing/pull/1866#discussion_r338539462